src/views/migrationpolicies/list/components/MigrationPolicyCreateForm/copmonents/MigrationPolicyFormDescription/MigrationPolicyFormDescription.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Text, TextVariants } from '@patternfly/react-core';
+
+const MigrationPolicyFormDescription: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <>
+      <Text component={TextVariants.p}>
+        {t(
+          'MigrationPolicy help you differentiate between various workloads. Adding MigrationPolicy',
+        )}
+      </Text>
+      <Text component={TextVariants.p}>
+        {t('will allow you to set priorities and security segregation per each workload.')}
```

Comment:
Can you please explain the reason of having the text divided like this? IMHO it's better not to divide it but to place it in the component of the correct dimensions or to set the dimensions if you wanted just to put the part of the text to a new line.

Reply:
Hi :) 
I removed one Text component to allow the `MigrationPolicyFormDescription` to get the width properties from the form of 50% (since in the doc there's no actual size I think that should do fine)
Thanks! :)